ci/build.ps1

```diff
@@ -116,6 +130,8 @@ if (-not $NoTests) {
     exit 1
   }
 
+  cpanm.bat -n Neovim::Ext
+  perl -W -e 'use Neovim::Ext; print $Neovim::Ext::VERSION'
```

Comment:
perhaps this block should live near the npm/ruby stuff (line ~116)

Reply:
It's the same install order as in `ci/install.sh`. I don't want to move it up because perl modules are often compiled. If anything, I may move it down if I need more OS packages.